### PR TITLE
feat(note-settings)/invite-collaborator-field

### PR DESCRIPTION
**Pull request by kloV148**
- Improve field, that shows join link
- Add copy button
- Add title to team section

Снимок экрана 2024-07-04 в 00 59 59

### CODE REVIEW

File: src/presentation/pages/NoteSettings.vue [hunk 1]
```diff
@@ -254,6 +237,17 @@ onMounted(async () => {
...
+
+.fieldset{
+  display: flex;
+  flex-direction: column;
+  gap: var(--spacing-xl);
+}
 
+.delete-button{
+  width: auto; /* Или конкретная ширина, например, 200px */
+  align-self: flex-start; /* Для выравнивания кнопки в начале контейнера */
```
Comment (slaveeks) [inline src/presentation/pages/NoteSettings.vue:250]: use english

**Review by slaveeks — APPROVED** [review]

File: src/presentation/components/noteSettings/InviteLink.vue [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+const invitationLink = computed(
+  () => `${import.meta.env.VITE_PRODUCTION_HOSTNAME}/join/${noteSettings.value?.invitationHash}`
+);
+
+/**
+ * Regenerate invitation hash
+ */
+async function regenerateHash() {
+  revokeHash(props.id);
```
Comment (neSpecc) [inline src/presentation/components/noteSettings/InviteLink.vue:56]: add confirm() call pls

File: src/presentation/components/noteSettings/InviteLink.vue [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+onMounted(async () => {
+  await loadSettings(props.id);
```
Comment (neSpecc) [inline src/presentation/components/noteSettings/InviteLink.vue:60]: unexpected call, you already have settings in parent component

File: src/presentation/pages/NoteSettings.vue [hunk 4]
```diff
@@ -61,34 +61,27 @@
...
-        >
-          {{ t('noteSettings.deleteNote') }}
-        </Button>
-      </div>
+
+      <Fieldset
+        :title="t('noteSettings.teamFormFieldSetTitle')"
+      >
+        <div class="fieldset">
+          <Team
```
Comment (neSpecc) [inline src/presentation/pages/NoteSettings.vue:69]: use large dimensions

**Review by elizachi — COMMENTED** [review]
When we revoke hash we can see alert about it, maybe add vessage window when we press on "Copy" button?

**Comment by kloV148** [comment]
> When we revoke hash we can see alert about it, maybe add vessage window when we press on "Copy" button?

I think it will be redundant, because we already have confirmation pop up when you're trying to revoke link. 
If something goes wrong, the user will know

File: src/presentation/components/noteSettings/InviteLink.vue [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+import type { NoteId } from '@/domain/entities/Note';
+import { useI18n } from 'vue-i18n';
+import { Button, Row, Section } from 'codex-ui/vue';
+import NoteSettings from '@/domain/entities/NoteSettings';
+
+const { t } = useI18n();
+
+const props = defineProps<{
+  id: NoteId;
+  noteSettings: NoteSettings;
```
Comment (neSpecc) [inline src/presentation/components/noteSettings/InviteLink.vue:44]: just pass `inviteHash`

Comment (kloV148) [reply src/presentation/components/noteSettings/InviteLink.vue]: Done

**Review by elizachi — APPROVED** [review]

**Review by neSpecc — APPROVED** [review]

**Pull request merged**